lib/active_storage.ex

```diff
@@ -0,0 +1,257 @@
+defmodule ActiveStorage do
+  @moduledoc """
+  The ActiveStorage context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Chaskiq.Repo
+
+  alias ActiveStorage.Blob
+
+  def verifier do
+    Chaskiq.Verifier
+  end
+
+  def graphql_resolver(
+        _a,
+        %{
+          input:
+            %{
+              byte_size: _byte_size,
+              checksum: _checksum,
+              content_type: _content_type,
+              filename: _filename
+            } = input
+        },
+        %{context: %{current_user: _current_user}}
+      ) do
+    direct_upload_response = %{direct_upload: create_direct_upload(input)}
+
+    {:ok, direct_upload_response}
+  end
```

Comment:
Maybe this should be pulled out into `chaskiq`?

Reply:
Ah, I get it now.  It's an integration with absinthe.  We might rename it to `absinthe_resolver`.  Or we could just put the code for the resolver in the `README`.  That way apps could use it / modify it themselves.

Reply:
yeah, this could be just documented on "Absinthe integration" 👍